plugins/Signatures/class.signatures.plugin.php

```diff
@@ -228,21 +228,33 @@ public function crossCheckSignature($Values, &$Sender) {
     }
 
     /**
-     * Checks signature length against Plugins.Signatures.MaxLength
+     * Determine if signature exceeds configured character limit.
      *
-     * @param $Values Signature settings form values
-     * @param $Sender Controller
+     * @param array $fields Signature settings form values
+     * @param ProfileController $sender
      */
-    public function checkSignatureLength($Values, &$Sender) {
-
+    public function checkSignatureLength($fields, &$sender) {
+        if (!property_exists($sender, 'Form') || !($sender->Form instanceof Gdn_Form)) {
+            return;
+        }
 
-        if (!is_null(self::getMaximumTextLength())) {
-            $Sig = Gdn_Format::to($Values['Plugin.Signatures.Sig'], $Sender->Form->getFormValue('Format'));
-            $TextValue = html_entity_decode(trim(strip_tags($Sig)));
+        $maxLength = self::getMaximumTextLength();
+        if ($maxLength !== null && $maxLength > 0) {
+            $maxLength = intval($maxLength);
+            $format = config('Plugin.Signatures.Format', Gdn_Format::defaultFormat());
```

Comment:
Ok but make sure to create an issue about #480 not having properly migrated all config values.

It migrated `Plugins.Signatures` but not `Plugin.Signatures`...